internal/trace/init.go

```diff
@@ -0,0 +1,12 @@
+package trace
+
+type URLRenderer func(traceID string) string
+
+var urlRenderer URLRenderer
+
+// Init initializes the trace package with a URL renderer. The passed function
+// will be used to take a traceID and turn it into an environment specific URL
+// that can be used to make debugging a little easier.
+func Init(ur URLRenderer) {
```

Comment:
If we want to break free from the `Init` convention, we can do a "register logger"-style setup where its expected consumers of this package can change the callback whenever they want to enable/disable behaviour

```suggestion
func RegisterURLRenderer(ur URLRenderer) {
```